### ISSUE TITLE
out of loom on boot

### ISSUE THREAD

**Issue by MnkyPwz** [urbit/urbit#2907]
I'm currently able to successfully boot up a comet but anytime I've attempted to boot my planet, I've greeted by this error. Any assistance in regarding to this would be incredibly helpful.

 ~/urbit-v0.10.4-darwin  ./urbit urbit-name
 2.19 L  INSERT
~
urbit 0.10.4
boot: home is /Users/usr/urbit-v0.10.4-darwin/urbit-name
loom: mapped 2048MB
lite: arvo formula 56587b84
lite: core 715fdde3
lite: final state 715fdde3
loom: mapped 2048MB
boot: protected loom
live: loaded: MB/406.716.416
boot: installed 262 jets
work: play 1024060
ames: live on 0
address 0x29322200a out of loom!
loom: [0x200000000 : 0x280000000)
Assertion '0' failed in noun/events.c:129
 
bail: oops
bailing out
Assertion failed: (0), function u3m_bail, file noun/manage.c, line 667.
[1] 36026 abort 
./urbit-name
%work: fail: broken pipe

**Comment by MnkyPwz** [comment]
By chance, does anyone happen to have any feedback and or suggestions relating to this error?

**Comment by Fang-** [comment]
Sorry to leave you hanging @MnkyPwz! I'm no expert on the underlying mechanics here, but from looking at a couple similar issues:

Did you maybe run out of disk space while your planet was running? Does it some disk space available now?

I've seen "replay your event log" recommended as a fix for similar symptoms. I can't make any guarantees that it'll fix the issue, but could be worth a try, especially if the planet hadn't been up for very long yet. You can move the `urbit-name/.urb/chk` folder to `urbit-name/.urb/chk-bak`
 or some other backup location, and then boot with `./urbit urbit-name` again. It should tell you "replaying events 0 through [some larger number]".

Tagging @joemfb in case he has further clarification/ideas here.

Also important to ask I guess: did your planet _ever_ boot successfully?

**Comment by MnkyPwz** [comment]
@Fang- thanks for your response. In fact, I did indeed run out of storage on my disk recently. So as you suspect, that very well might be my issue. I'll move some files onto an external hard drive, free up some space and give it another attempt. Hoping that solves it.

Also, yes, I've been able to boot up for the last few months. This was the first time I experienced anything like this, which is mostly why I was surprised it stopped booting.

I'll give it a go and circle back here. Thank you again for taking the time to get back to me.
